docs/fundamentals/diagnostics/runtime-tiered-compilation-events.md

```diff
@@ -0,0 +1,117 @@
+
+## TieredCompilationSettings event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationSettings`|280|Provides information about tiered compilation settings.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+|`Flags`|`win:UInt32`|Flags that indicate various settings. Multiple flags may be provided with bitwise-OR, and a value of `0x0` indicates that no flags were provided.<br/><br/>`0x1` - Quick JIT is enabled. If a method does not have precompiled code, for the first tier it will be JIT-compiled quickly but with fewer optimizations.<br/><br/>`0x2` - Quick JIT is enabled for methods containing loops.<br/><br/>`0x4` - Tiered PGO is enabled. Methods may be profiled before they are optimized.<br/><br/>`0x8` - ReadyToRun is enabled. Methods that have precompiled ReadyToRun code will use that for the first tier.|
+
+## TieredCompilationPause event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationPause`|281|Tiered compilation was paused. Tiered compilation may be paused due to startup-like activities such as a new method being called for the first time, to reduce overhead during application startup, such as call-counting and background JIT-compilation. It would resume after startup-like activities cease for a short duration.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+
+## TieredCompilationResume event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
+
+The following table shows the event information.
+
+|Event|Event ID|Raised when|
+|-|-|-|
+|`TieredCompilationResume`|282|Tiered compilation was resumed.|
+
+The following table shows the event data.
+
+|Field name|Data type|Description|
+|-|-|-|
+|`ClrInstanceID`|`win:UInt16`|Unique ID for the instance of CoreCLR.|
+|`NewMethodCount`|`win:UInt32`|The number of new methods called for the first time while tiered compilation was paused.|
+
+## TieredCompilationBackgroundJitStart event
+
+The following table shows the keyword and level.
+
+|Keyword for raising the event|Level|
+|-|-|
+|`CompilationKeyword` (0x1000000000)|Informational (4)|
```

Comment:
Is this the same for all the events? Should we just list it once at the beginning of the article?

Reply:
The info here is the same for all of the events in this article, though there are differences between events in other cases. Also typically, a user would look in the docs for a particular event and it would be easier to follow if all of the relevant information about the event is in one place.